### DISCUSSION TITLE
Don't restore pagination (& page size) in presets

### DISCUSSION THREAD

**Discussion by domdus** [directus/directus#14477] [Feature Requests]
Editors use global presets I created and shared for them.
When you use the pagination in a preset to navigate to e.g. page 30 - edit some item on detail page and then return or save changes - the preset is initialized on page 1 again.

It would be great to have a preset option to not store or 
actually load the stored page number and page size again to give user the freedom to navigate as they need to.

**Comment by benhaynes** [comment]
We should keep the page count, I think it's useful (depending on layout) to set/save page count. Maps might want 1k, other heavy pages might want 25.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
